### PR TITLE
Configure the policy machine to use Travis

### PR DESCRIPTION
**Pull request by DMcKinnon-mdsol**
This PR configures a Travis build for the policy machine gem. Seems to be past time.

@mdsol/team-04 please review.

### CODE REVIEW

**Comment by DMcKinnon-mdsol** [comment]
Whelp, open source repos aren't billable. Adios Travis.

**Comment by DMcKinnon-mdsol** [comment]
Reopened because there are other ways to Travis.

**Comment by DMcKinnon-mdsol** [comment]
Woo!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+sudo: required
```
Comment (CGDS) [inline .travis.yml]: I think travis made pg 9.5 available on trusty containers a couple of months ago, dont need this and the before_script anymore, methinks

**Pull request merged**